### ISSUE TITLE
Make s3 plugin more configurable for bulk uploading.

### ISSUE THREAD

**Issue by beenanner** [sitespeedio/sitespeed.io#1456]
Currently there appears to be an issue if you have too many files being uploaded at the same time..

```
[2017-02-15 04:53:56] Could not upload to S3 Error: ETag does not match MD5 checksum
    at /usr/src/app/node_modules/s3/lib/index.js:410:16
    at Pend.wait (/usr/src/app/node_modules/pend/index.js:21:5)
    at Response.<anonymous> (/usr/src/app/node_modules/s3/lib/index.js:407:14)
    at Request.<anonymous> (/usr/src/app/node_modules/aws-sdk/lib/request.js:350:18)
    at Request.callListeners (/usr/src/app/node_modules/aws-sdk/lib/sequential_executor.js:100:18)
    at Request.emit (/usr/src/app/node_modules/aws-sdk/lib/sequential_executor.js:77:10)
    at Request.emit (/usr/src/app/node_modules/aws-sdk/lib/request.js:604:14)
    at Request.transition (/usr/src/app/node_modules/aws-sdk/lib/request.js:21:12)
    at AcceptorStateMachine.runTo (/usr/src/app/node_
```

**Comment by soulgalore** [comment]
released in 4.4.1, lets see if it helps.

**Comment by soulgalore** [comment]
done
